src/taipy/core/config/config.schema.json

```diff
@@ -408,8 +412,8 @@
             }
           },
           "sequences": {
-            "description": "The dictionary of sequence descriptions.",
-            "type": "object",
+            "description": "The list of task configs.",
+            "type": "array",
```

Comment:
I think `sequences` is an object with properties that are string arrays ?

Reply:
Not sure if I understand correctly, have made changes to it

Reply:
I think the right way of describing it is:
```
"sequences": {
            "description": "The dictionary of sequence descriptions.",
            "type": "object",
            "additionalProperties": {"type": "array", "items": {"type": "string", "uniqueItems": true}}
          }
```

so that we support a TOML like

```
[SCENARIO.mon_scenario.sequences]
   seq1= ["hh", "hh"]
   seq2= ...

```
or

```
[SCENARIO.mon_scenario]
...
sequences={
   seq1= ["hh", "hh"]
   seq2= ...
   }
```

Reply:
I'm not an expert in this so I'll just follow what the expert tells me to do 😃

Reply:
I'm no expert either :-)
https://json-schema.org/understanding-json-schema/reference/object.html